cmd/sys-mgmt-agent/res/manifest.toml

```diff
@@ -0,0 +1,17 @@
+# This TOML manifest file specifies the services managed by the SMA.
+# The bootstrap property (like –manifest) will indicate to the SMA to get this TOML configuration from the file system.
+# The SMA needs the details provided in this TOML to stop/start/restart each service that the SMA manages.
+# What about specifying the edgexsecurity service to use?
+# Different versions of this manifest may exist depending on how/where EdgeX is deployed (Docker v. Snappy, Windows v. Linux, etc.).
+# In the first release of the SMA, the manifest will be static.
+# In the future, the manifest may be more dynamic or even provided by some 3rd party orchestrator.
+
+# title = 'EdgeX manifest for consumption by the SMA to bootstrap managed services.'
+
+# edgex-support-notifications
+EdgexServicesNotificationsName = 'notifications'
+EdgexServicesNotificationsHost = 'localhost'
+EdgexServicesNotificationsPort = 48060
+EdgexServicesNotificationsProtocol = 'http'
+EdgexServicesNotificationsCommandstop = 'stop'
+EdgexServicesNotificationsCommandstart = 'start'
```

Comment:
Utilize the hierarchical capabilities of TOML to organize the content of this file, esepcially as it grows.
```
[Services]
    [Services.Notifications]
    Name = 'notifications'
    Host   = 'localhost'
    Port    = 48060
...
```
As an aside, instead of "Notifications" you could use the value of the service key "edgex-support-notifications" if you think that would be cleaner in the code. This will make the "Name" property redundant BTW.

Does this need to be in a separate manifest.toml file or can it be included in configuration.toml? You will need both a local and docker-ized version of this information anyhow.